packages/bpk-component-barchart/src/BpkBarchartBar.js

```diff
@@ -51,35 +51,53 @@ const BpkBarchartBar = (props) => {
     onClick,
     outlier,
     selected,
+    padding,
```

Comment:
Nope. Well, kind of. It doesn't really make sense to use it outside of BpkBarchart, but who knows? We export it, so we should probably document it. What do you think @matthewdavidson ? Also, with all the other components in the barchart...